docs/core/diagnostics/dotnet-trace.md

```diff
@@ -180,6 +185,42 @@ To collect traces using `dotnet-trace`:
 
 - Stop collection by pressing the `<Enter>` key. `dotnet-trace` will finish logging events to the *trace.nettrace* file.
 
+## Launch a child application and collect a trace from its startup using dotnet-trace
+
+NOTE: This works for apps running .NET 5.0 or later only.
```

Comment:
This disclaimer should either be also in .NET counters or just the mention in the parameter documentation. Given how common it is to just look at the examples, I'd say the first option is better..